manager/src/test/java/com/redhat/service/bridge/manager/ProcessorServiceTest.java

```diff
@@ -366,12 +376,9 @@ public void testGetProcessorsCount() {
     @Test
     public void testDeleteProcessor() {
```

Comment:
You could add a test for the the `Processor` is `FAILED` too.

Reply:
That is when it is `PROVISIONING` not `FAILED`. When `FAILED` you should be allowed to delete it, but I don't see a test for that... however please point me to one. I could not see it.

Reply:
okies, I miss took your earlier comment. I thought you are asking to add failure usecase. I will add it.

Reply:
I had new testcase for `FAILED` status
https://github.com/vaibhavjainwiz/sandbox/blob/f68de90bd03dc5bf56f0059025d09312f694e6b6/manager/src/test/java/com/redhat/service/bridge/manager/ProcessorServiceTest.java#L393